ui/pages/confirmation/templates/switch-ethereum-chain.js

```diff
@@ -68,9 +68,18 @@ function getValues(pendingApproval, t, actions) {
           element: 'ConfirmationNetworkSwitch',
           key: 'network-being-switched',
           props: {
-            newNetwork: {
-              chainId: pendingApproval.requestData.chainId,
-              nickname: pendingApproval.requestData.nickname,
+            toNetwork: {
+              chainId:
+                pendingApproval.requestData.toNetworkConfiguration.chainId,
+              nickname:
+                pendingApproval.requestData.toNetworkConfiguration.nickname,
```

Comment:
does `toNetwork` need `type`?

Reply:
Hmm yeah I'd think so... But also, maybe this should just be 
```
toNetwork: pendingApproval.requestData.toNetworkConfiguration,
...
```